jsonschema-core.xml

```diff
@@ -313,43 +313,70 @@
                 </t>
                 <section title="JSON Schema Objects and Keywords">
                     <t>
-                        Object properties that are applied to the instance are called keywords,
-                        or schema keywords.  Broadly speaking, keywords fall into one
-                        of five categories:
+                        Object properties that are applied to the instance are called keywords, or schema keywords.
+                        Two properties in the same schema object are referred to as adjacent keywords.
+                        As an authoring convenience, adjacent keywords can sometimes affect each other in a way that a property from a different schema (including sub-schemas) cannot.
+                    </t>
+                    <t>
+                        Keywords are classified into one or more of six categories:
                     </t>
                     <dl>
-                        <dt>identifiers</dt>
+                        <dt>Core keywords</dt>
+                        <dd>
+                            Keywords that may configure how the schema is processed,
+                            and can modify the meaning of other keywords in the schema.
```

Comment:
What kind of core keyword can modify the meaning of another keyword?

There's maybe `$vocabulary`, but this _defines_ what the keywords mean; it doesn't modify them.